tests/features/lru_cache_test.py

```diff
@@ -79,6 +81,52 @@ def test_fix_no_arg_decorators_noop(s, min_version):
         ),
     ),
 )
-def test_fix_no_arg_decorators(s, expected):
-    ret = _fix_plugins(s, settings=Settings(min_version=(3, 8)))
+def test_fix_no_arg_decorators(s, min_version, expected):
+    ret = _fix_plugins(s, settings=Settings(min_version=min_version))
+    assert ret == expected
+
+
+@pytest.mark.parametrize(
+    ('s', 'min_version'),
+    (
+        pytest.param(
+            'from functools import lru_cache\n'
+            '@lru_cache(maxsize=None)\n'
+            'def foo(): pass\n',
+            (3, 9),
+            id='pyupgrade doesn\'t add or remove imports '
```

Comment:
don't add unnecessary escape sequences

also this id is unnecessarily verbose, "from imported" would be sufficient